remix.config.js

```diff
@@ -1,22 +1,9 @@
 /** @type {import('@remix-run/dev').AppConfig} */
 module.exports = {
-  devServerBroadcastDelay: 1000,
   ignoredRouteFiles: ["**/.*"],
-  server: "./server.js",
-  serverBuildPath: "functions/[[path]].js",
-  serverConditions: ["worker"],
-  serverDependenciesToBundle: "all",
-  serverMainFields: ["browser", "module", "main"],
-  serverMinify: true,
-  serverModuleFormat: "esm",
-  serverPlatform: "neutral",
-  // appDirectory: "app",
-  // assetsBuildDirectory: "public/build",
-  // publicPath: "/build/",
   future: {
     unstable_postcss: true,
     unstable_tailwind: true,
-    unstable_dev: true,
```

Comment:
Need to research this one a bit further, this is supposed to be the new version of the dev server, but with this the node app doesn't run.